src/common/hooks/useOutsideAlerter.ts

```diff
@@ -0,0 +1,15 @@
+import { RefObject, useEffect } from 'react';
```

Comment:
Is this one from here :) ? https://usehooks.com/useOnClickOutside/

Reply:
I think I found it elsewhere